Assignment4/q1-accept_anything.go

```diff
@@ -0,0 +1,72 @@
+/*
+
+	1. The given program accepts an integer value between 1 to 4 from the user.
+	There is an option associated with each value, which is basically objects of different data types with some associated value.
+
+	Write a method named "AcceptAnything" which takes any type of data as input.
+
+	Based on the option chosen by the user, we will send different types of objects to this function and it should print the following based on its respective data type of value sent to the function "AcceptAnything".
+
+	integer :
+	"This is a value of type Integer, <value>"
+
+	string :
+	"This is a value of type String, <value>"
+
+	boolean :
+	"This is a value of type Boolean, <value>"
+
+	Custom data type Hello :
+	"This is a value of type Hello, <value>"
+*/
+
+package main
+
+import "fmt"
+
+type Student struct {
```

Comment:
If we aren't exporting any variable to another module, then it should be in small-case camel-casing

Reply:
done